addons/website_slides/models/slide_slide.py

```diff
@@ -533,6 +534,13 @@ def _compute_website_url(self):
                 base_url = slide.channel_id.get_base_url()
                 slide.website_url = '%s/slides/slide/%s' % (base_url, slug(slide))
 
+    @api.depends('channel_id.website_id.domain')
```

Comment:
Strange depends ... as this is anyway not stored I don't think we have to go that far. I would simply add a "false" depends on is_published maybe, even if actually this field will not really have its value depending on any field. It is better to always have a trigger otherwise computation is not always granted.